PayPalMessaging/src/main/java/com/braintreepayments/api/PayPalMessagingView.kt

```diff
@@ -43,19 +51,15 @@ class PayPalMessagingView(
                     )
                     notifyFailure(error = clientIdError)
                 } else {
-                    val payPalMessageView = constructPayPalMessageView(context, clientId, configuration, request)
+                    val payPalMessageView =
+                        constructPayPalMessageView(context, clientId, configuration, request)
                     payPalMessageView.layoutParams = ViewGroup.LayoutParams(
                         ViewGroup.LayoutParams.MATCH_PARENT,
                         ViewGroup.LayoutParams.MATCH_PARENT
                     )
 
                     addView(payPalMessageView)
                 }
-            } else {
-                val unknownError = BraintreeException(
-                    "Fetching Braintree configuration resulted in no error or configuration returned."
-                )
-                notifyFailure(error = unknownError)
```

Comment:
Yeah this is a good catch. Technically it won't, but this code is good for completeness since it's theoretically possible for both values to be null (maybe through a bug of some sort).

It's definitely a code smell. Our callback pattern also has Pre-Kotlin roots. Ideally we'll move to a `sealed class Result` type that behaves similar to Swift's `Result` type that will allow us to guarantee at compile time that a `(null, null)` callback scenario never happens.